app/views/partials/lib/commands/emails/auth-link.liquid

```diff
@@ -0,0 +1,25 @@
+---
+metadata:
```

Comment:
What is the purpose of this?

Reply:
a partial as a function and it is an interface of the function

in future we could do static analyses and automatic tests